hio_stream_target.py

```diff
@@ -22,5 +34,9 @@ def send_message(self, image_bytes, image_file_name, metadata):
         # TODO: add link!
         pickled_metadata = bytearray(pickle.dumps(metadata))
 
-        message = pickled_metadata + image_bytes
-        self.sc.send_data(message)
+        message_bytes = pickled_metadata + image_bytes
+
+        # TODO: issue here - only 1 arg?
```

Comment:
1 arg here:
https://github.com/beirbear/HarmonicSC/blob/master/stream_connector.py
3 args in ST's example.